tests/glsl/shader-subgroup-shuffle-relative.slang

```diff
@@ -0,0 +1,65 @@
+//TEST:SIMPLE(filecheck=CHECK_GLSL):  -allow-glsl -stage compute -entry computeMain -target glsl
+//TEST:SIMPLE(filecheck=CHECK_SPV):  -allow-glsl -stage compute -entry computeMain -target spirv
+
+// not testing hlsl due to missing impl
+// TEST:SIMPLE(filecheck=CHECK_HLSL): -allow-glsl -stage compute -entry computeMain -target hlsl
+// not testing cuda due to missing impl
+// TEST:SIMPLE(filecheck=CHECK_CUDA): -allow-glsl -stage compute -entry computeMain -target cuda 
+// not testing cpp due to missing impl
+// TEST:SIMPLE(filecheck=CHECK_CPP):  -allow-glsl -stage compute -entry computeMain -target cpp
+
+//TEST(compute, vulkan):COMPARE_COMPUTE(filecheck-buffer=BUF):-vk -compute -entry computeMain -allow-glsl
+//TEST(compute, vulkan):COMPARE_COMPUTE(filecheck-buffer=BUF):-vk -compute -entry computeMain -allow-glsl -emit-spirv-directly
+
+//#define TEST_when_doubles_dont_crash
```

Comment:
If this should be applied only to SPIR-V target, you should wrap this like
```
#if !defined(TARGET_SPIRV)
#define TEST_when_doubles_dont_crash
#endif
```

Reply:
it crashes both

Reply:
I said it on my previous comments, but repeating here.
The syntax of this test file is in GLSL and it doesn't make sense if the test fails for targeting GLSL.
If GLSL fails, you have to debug and figure out what is going on.